script/security-check.js

```diff
@@ -11,6 +11,7 @@ const packageJSON = require('../package.json');
 const exceptionSet = new Set([
   'https://npmjs.com/advisories/996',
   'https://npmjs.com/advisories/1488',
+  'https://npmjs.com/advisories/1654',
```

Comment:
This was merged in from https://github.com/department-of-veterans-affairs/vets-website/pull/16565 so that we don't have to wait ~45 mins for that branch to be merged before CI in this branch will pass.